client/state/billing-transactions/types.ts

```diff
@@ -0,0 +1,103 @@
+import type { RawReceiptPurchase } from 'calypso/state/receipts/types';
+
+export interface IndividualReceipt {
+	receipt_id: number;
+	purchases: RawReceiptPurchase[];
+	display_price: string;
+	price_integer: number;
+	price_float: number;
+	currency: string;
+	is_gift_purchase: boolean;
+}
+
+export interface BillingTransaction {
+	id: string;
+	service: string;
+	amount: string;
+	tax_amount: string;
+	icon: string;
+	date: string;
+	desc: string;
+	org: string;
+	url: string;
+	support: string;
+	pay_ref: string;
+	pay_part: string;
+	cc_type: string;
+	cc_num: string;
+	cc_name: string;
+	cc_email: string;
+	credit: string;
+	items: BillingTransactionItem[];
+}
+
+export interface BillingTransactionItem {
+	id: string;
+	type: string;
+	type_localized: string;
+	domain: string;
+	site_id: string;
```

Comment:
It might be odd that the site ID is a string.

Reply:
I agree, but that's what the endpoint returns:

<img width="234" alt="Screenshot 2023-02-17 at 1 48 20 PM" src="https://user-images.githubusercontent.com/2036909/219758465-32e69fd1-486b-477a-af3d-63d6fbe1751b.png">